Implementation/FindMyBLEDevice/FindMyBLEDevice/ViewModels/StrengthViewModel.cs

```diff
@@ -56,15 +65,49 @@ public int CurrentRssi
             }
         }
 
-        public async void OnAppearing()
+        public string Status
         {
-            /// works since database id counts from 1 - might change to nullable int
-            if (!(App.DevicesStore.SelectedDevice is null))
+            get => _status;
+            set => SetProperty(ref _status, value);
+        }
+
+        public void OnAppearing()
+        {
+            if (App.DevicesStore.SelectedDevice is null)
             {
-                await App.Bluetooth.StartRssiPolling(App.DevicesStore.SelectedDevice.BT_GUID, (int v) =>
+                Status = "No device selected!\nPlease select a device to continue.";
+            }
+            else
+            {
+                Status = "Connecting to \"" + App.DevicesStore.SelectedDevice.UserLabel + "\"...\n" +
```

Comment:
Very nice solution!
I think we should change this code such that UI-Texts are not directly in our source code, but rather use a `ResourceManager` of some sorts.